src/models/decisions/decision.ts

```diff
@@ -0,0 +1,17 @@
+import { EnumValue } from "../../helpers/advancedTypes";
+
+export enum VerificationDecision {
+    FALSE = "false",
+    TRUE = "true",
+    UNSURE = "unsure",
+    SKIP = "skip",
+}
+
+// A polymorphic structure that can be used to represent any decision type
+// that implements the interface
+export interface IDecision {
+    confirmed: EnumValue<VerificationDecision>;
+    identifier: symbol;
+    type: string;
+    colorIndex: number;
+}
```

Comment:
Seems like an abstract class would have offered the same flexibility with less verbosity.